packages/scripts/README.md

```diff
@@ -65,22 +66,31 @@ A preset is a node module `talend-scripts-preset-${presetName}`. It exports 3 fu
 
 ```javascript
 module.exports = {
-	getWebpackConfiguration(presetApi) {},
-	getJestConfiguration(presetApi) {}, // TODO
-	getEslintConfiguration(presetApi) {} // TODO
+	getWebpackConfiguration(presetApi) { },
+	getEslintConfiguration(presetApi) { },
 }
 ```
+| Preset function | Description |
+|---|---|
+| getWebpackConfiguration | Returns the webpack configuration object. |
+| getEslintConfiguration | Returns the path to .eslintrc file. It will be passed to eslint --config option. This path should be absolute or relative from cwd. |
```

Comment:
here it's a path and it's an object for Webpack? 🤔 What do you think of `getEslintConfigurationPath` instead?